arbnode/batch_poster.go

```diff
@@ -369,7 +369,7 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context, timeSinceBatc
 	}
 
 	if b.das != nil {
-		cert, err := b.das.Store(ctx, sequencerMsg)
+		cert, err := b.das.Store(ctx, sequencerMsg, das.CALLEE_PICKS_TIMEOUT)
```

Comment:
Let's discuss whether the batch poster should choose the timeout here, versus letting the callee (DAS or aggregator) choose. The question is which layer of the system is best positioned to know what is required for the end-to-end needs of the protocol.

Reply:
Either the batch poster or aggregator needs to choose. Having the batch poster pick is nice because it avoids the special value CALLEE_PICKS_TIMEOUT, so I am slightly leaning towards that.

Reply:
Yes, I agree.